docs/README.md

```diff
@@ -27,42 +27,33 @@ You can download the installer for the *Recommended Build* or the
  
 For support and questions, visit [the NeoForged Discord server][Discord].
 
-# Creating Mods
+## Creating Mods
 
 [See the "Getting Started" section in the NeoForged Documentation][Getting-Started].
 
-# Contribute to Forge
+## Contribute to NeoForge
 
-If you wish to actually inspect Forge, submit PRs or otherwise work
- with Forge itself, you're in the right place!
-
- [See the guide to setting up a Forge workspace][ForgeDev].
+If you wish to actually inspect NeoForge, submit PRs or otherwise work
+with NeoForge itself, you're in the right place!
 
 ### Pull requests
 
-[See the "Making Changes and Pull Requests" section in the Forge documentation][Pull-Requests].
-
 Please read the contributing guidelines found [here][Contributing] before making a pull request.
 
 ### Contributor License Agreement
-We require all contributors to acknowledge the [Neoforged Contributor License Agreement][CLA]. 
+We require all contributors to acknowledge the [NeoForged Contributor License Agreement][CLA]. 
 Please ensure you have a valid email address associated with your GitHub account to do this. If you have previously 
  signed it, you should be OK.
 
 #### Donate
 *NeoForged is a large project with many collaborators working on it around the clock. It will always remain 
```

Comment:
This section needs adjusting too, we don't accept donations at the moment.

Reply:
Oh, I see. I will fix that now.

Reply:
This issue is fixed in this commit (https://github.com/neoforged/NeoForge/pull/232/commits/78254a0755351d14d38b3145e34acb09c168e2a8).